pkgs/development/python-modules/pylatex/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, matplotlib
+, numpy
+, ordered-set
+, quantities
+}:
+
+buildPythonPackage rec {
+  pname = "pylatex";
+  version = "1.3.4";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "JelteF";
+    repo = "PyLaTeX";
+    rev = "v${version}";
+    hash = "sha256-7JE8xKryf7sjvlUjDH7jUMm7CyirjPbmWo+reWeGIlc=";
+  };
+
+  passthru.optional-dependencies = {
+    withMatplotlib = [ matplotlib ];
+    withMatrices = [ numpy ];
+    withQuantities = [ numpy quantities ];
```

Comment:
Names should mach the corresponding extras [here](https://github.com/JelteF/PyLaTeX/blob/2050b14d8d7ed4fe788c769afec6816e2b703355/setup.py#L30-L37)

Reply:
> Names should mach the corresponding extras [here](https://github.com/JelteF/PyLaTeX/blob/2050b14d8d7ed4fe788c769afec6816e2b703355/setup.py#L30-L37)

Should be fixed now 👍